.buildkite/pipeline-utils/ci-stats/pick_test_group_run_order.ts

```diff
@@ -124,15 +127,53 @@ function isObj(x: unknown): x is Record<string, unknown> {
   return typeof x === 'object' && x !== null;
 }
 
+interface PartialConfigs {
+  enabled?: string[];
+  disabled?: string[];
+  defaultQueue?: string;
+}
+
 function getEnabledFtrConfigs(patterns?: string[]) {
+  const configs: {
+    enabled: Array<string | { [configPath: string]: { queue: string } }>;
+    uniqueQueues: Set<string>;
+    defaultQueue?: string;
+  } = {
+    enabled: [],
+    uniqueQueues: new Set<string>(),
+  };
+  for (const manifestRelPath of ALL_FTR_MANIFEST_REL_PATHS) {
+    try {
+      const partialConfigs: PartialConfigs = loadYaml(
+        Fs.readFileSync(manifestRelPath, 'utf8')
+      ) as PartialConfigs;
+
+      configs.enabled = configs.enabled.concat(partialConfigs?.enabled ?? []);
+      if (partialConfigs?.defaultQueue) {
+        configs.uniqueQueues.add(partialConfigs.defaultQueue);
+      }
+    } catch (_) {
+      const error = _ instanceof Error ? _ : new Error(`${_} thrown`);
+      throw new Error(`unable to parse ${manifestRelPath} file: ${error.message}`);
+    }
+  }
+
   try {
-    const configs = loadYaml(Fs.readFileSync('.buildkite/ftr_configs.yml', 'utf8'));
     if (!isObj(configs)) {
       throw new Error('expected yaml file to parse to an object');
     }
     if (!configs.enabled) {
       throw new Error('expected yaml file to have an "enabled" key');
     }
+    if (configs.uniqueQueues.size !== 1) {
+      throw Error(
+        `FTR configs yml files should define the same 'defaultQueue': ${[
+          ...configs.uniqueQueues,
+        ].join(' ')}`
+      );
+    }
+    configs.defaultQueue = configs.uniqueQueues.values().next().value;
```

Comment:
I don't like it and config can have its own custom worker queue definition. Maybe we should defined `defaultQueue` for FTR tests directly in code to make sure it is the same across all manifests?